ui/app/templates/components/mount-backend-form.hbs

```diff
@@ -1,96 +1,124 @@
 <PageHeader as |p|>
   <p.levelLeft>
-    <h1 class="title is-3" data-test-mount-form-header=true>
+    <h1 class="title is-3" data-test-mount-form-header="true">
       {{#if showConfig}}
-        {{#with (find-by 'type' mountModel.type mountTypes) as |typeInfo|}}
-          <ICon @size=24 @glyph={{concat "enable/" (or typeInfo.glyph typeInfo.type)}} @class="has-text-grey-light" />
+        {{#with (find-by "type" mountModel.type mountTypes) as |typeInfo|}}
+          <ICon
+            @size="24"
+            @glyph={{concat "enable/" (or typeInfo.glyph typeInfo.type)}}
+            @class="has-text-grey-light"
+           />
+
           {{#if (eq mountType "auth")}}
-            Enable {{typeInfo.displayName}} authentication method
+            Enable{{typeInfo.displayName}}authentication method
```

Comment:
I think this broke the whitespace here and below...

Reply:
agh my linter does this. it's killing me bit by bit. I have to make everything a string